tests/allocator/allocator_test.py

```diff
@@ -57,11 +67,10 @@ def test_get_beneficiary_plan_params(skale_allocator):
 
 def test_get_plan(skale_allocator):
     plan = skale_allocator.allocator.get_plan(D_PLAN_ID)
-    assert plan == {
-        'totalVestingDuration': 36,
-        'vestingCliff': 6,
-        'vestingIntervalTimeUnit': 1,
-        'vestingInterval': 6,
-        'isDelegationAllowed': True,
-        'isTerminatable': True
-    }
+    assert plan == TEST_PLAN_PARAMS
+
+
+def test_get_all_plans(skale_allocator):
+    plans = skale_allocator.allocator.get_all_plans()
+    TEST_PLAN_PARAMS['planId'] = 1
```

Comment:
I think it's better to leave constants immutable.

Reply:
addressed